tests/unittests/annotation/validate_label_schema_tests.py

```
@@ -0,0 +1,795 @@
+
+        validate_field_label_schema(
+            dataset,
+            "str_list_field",
+            {
+                "type": "list<str>",
+                "component": "text",
+            },
+        )
+
+        # 'values' is not provided
+        with self.assertRaises(ValueError):
```

Ruff (0.14.7)

Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)